pkg/operator/ceph/object/controller.go

```diff
@@ -441,7 +441,9 @@ func (r *ReconcileCephObjectStore) reconcile(request reconcile.Request) (reconci
 	// CREATE/UPDATE
 	_, err = r.reconcileCreateObjectStore(cephObjectStore, request.NamespacedName, cephCluster.Spec)
 	if err != nil && kerrors.IsNotFound(err) {
-		logger.Info(opcontroller.OperatorNotInitializedMessage)
+		// A not found error may mean ceph is still initializing, but there might be some other error
+		// so we log the error and requeue
+		logger.Warningf("object store %q reconcile failed, ceph may still be initializing. %v", cephObjectStore.Name, err)
```

Comment:
Namespaced name will help distinguish between stores if there are multiple clusters with same-named stores.

```suggestion
		logger.Warningf("object store %q reconcile failed, ceph may still be initializing. %v", cephObjectStore.Namespace+"/"+cephObjectStore.Name, err)
```